docs/getting-started/deployment/infrastructure-as-code/overview.md

```diff
@@ -0,0 +1,21 @@
+# Infrastructure as Code
+
+CarbyneStack has adopted Infrastructure as Code (IaC) as a core principle. IaC is the process of managing and provisioning infrastructure through code instead of manually deploying resources via kubectl, helm, etc. This allows for the infrastructure to be versioned, tested, and deployed in a repeatable manner.
+
+## Terraform
+
+CarbyneStack uses [Terraform by Hashicorp](https://www.terraform.io/) to provision infrastructure and deploy CarbyneStack on top of it.  Terraform has a large community and is widely used in the industry, with support for most major cloud providers through a large set of [providers](https://registry.terraform.io/browse/providers?product_intent=terraform).
```

Comment:
Extra whitespace `... it.   Terraform`

Reply:
fix it and other instances - I was taught this from very early, so adding double-space after a period is a reflex at this point :)